test/integration/nicotine.robot

```diff
@@ -11,5 +11,5 @@ Library           nicotine_library.py
 
 Run nicotine as shell command
     Running nicotine starts a process    nicotine    3
-    Result should be    0
+    Result should be    ${True}
```

Comment:
Thanks @lene. Ignore my incompetence. I'm not very familiar with the Robot Framework.